lib/chef/knife/clc_server_create.rb

```diff
@@ -443,12 +458,19 @@ def execute
 
       def sync_create_server
         ui.info 'Requesting server launch...'
-        links = connection.create_server(prepare_launch_params)
+        launch_params = prepare_launch_params
+
+        if config[:clc_bootstrap] && config[:clc_source_server] =~ /WIN/ && os_linux?
```

Comment:
Well, `clc_source_server` might be actually an ID looking like this: `asdkjfasdhkbf14cadfsa412rc`. I don't think it's reliable way to check server platform. Take a look at `check_server_platform` method.